### DISCUSSION TITLE
Can't Uninstall - RaspAP Uninstall Error: Failed to disable raspap.service

### DISCUSSION THREAD

**Discussion by kave3** [RaspAP/raspap-webgui#1329] [Q&A]
Greetings,

I need to uninstall RaspAP. I have followed the uninstall commands provided in the FAQ: 
https://docs.raspap.com/faq/#how-do-i-uninstall-raspap

I get the following error...
```
pi@raspberrypi:/var/www/html $ sudo installers/uninstall.sh
RaspAP Uninstall: Configure uninstall of RaspAP
Detected Raspbian GNU/Linux 9.9 (stretch)
RaspAP install directory: /etc/raspap
Lighttpd install directory: /var/www/html? [Y/n]: y
Uninstall from lighttpd directory: /var/www/html
Uninstall RaspAP with these values? [Y/n]: y
Restore the last dhcpcd.conf file? [y/N]: y
RaspAP Uninstall: Removing raspapd.service
Failed to disable unit: No such file or directory
RaspAP Uninstall Error: Failed to disable raspap.service
```

How do I uninstall RaspAP and return my dhcpcd.conf file to its previous config?

**Comment by billz** [comment]
These FAQs address your questions (try the docs search): 

https://docs.raspap.com/faq/#raspap-service
https://docs.raspap.com/faq/#custom

The `raspapd.service` is an optional component. If you didn't install it there is nothing to uninstall.
Your original config files are located in `/etc/raspap/backups/`. The uninstaller will restore these files for you, unless you are using nonstandard file paths.
